tests/monday_types/location_type_tests.py

```diff
@@ -0,0 +1,138 @@
+    column_value =  en.cv.create_column_value(column_type,id=id,title=title,value=value)
+
+    # Act
+
+    location_type = t.LocationType(title='Location')
+    value = location_type.to_native(column_value)
+
+    # Assert
+    eq_(value.lat,26.930101)
+    eq_(value.lng,-80.790198)
+
+
+def test_should_succeed_when_to_native_returns_a_location_when_passed_an_import_dict_value():
+    
+    # Arrange
+
+    location_type = t.LocationType(title='Location')
+
+    # Act
+
+    value = location_type.to_native({'lat': 26.930101, 'lng': -80.790198, 'address': "Home address"})
+
+    # Assert
+    eq_(value.lat,26.930101)
+    eq_(value.lng,-80.790198)
+    
+
+@raises(ConversionError)
+def test_should_succeed_when_to_native_raises_a_conversionerror_when_passed_an_invalid_import_dict():
+    
+    # Arrange
+
+    location_type = t.LocationType(title='Location')
+
+    # Act
+
+    value = location_type.to_native({'latitude': 26.930101, 'longitude': -80.790198, 'address': "Home address"})
+
+
+def test_should_succeed_when_to_native_returns_none_when_passed_a_none():
+    
+    # Arrange
+
+    location_type = t.LocationType(title='Location')
+
+    # Act
+
+    value = location_type.to_native(None)
+
+    # Assert
+    eq_(value,None)
+
+
+def test_should_succeed_when_to_primitive_returns_empty_dict_when_passed_a_none():
+
+    # Arrange
+
+    location_type = t.LocationType(title='Location')
+
+    # Act
+
+    value = location_type.to_primitive(None)
+
+    # Assert
+    eq_(value,{})
+
+
+def test_should_succeed_when_to_primitive_returns_export_dict_when_passed_a_location_value_():
+    
+    # Arrange
+
+    location_type = t.LocationType(title='Location')
+
+    # Act
+
+    value = location_type.to_primitive(en.cv.Location(lat = 26.930101, lng= -80.790198, address = "Home address"))
+
+    # Assert
+    eq_(value['lat'],26.930101)
+    eq_(value['lng'],-80.790198)
+
+
+def test_should_succeed_when_to_primitive_returns_empty_dict_when_passed_a_location_value_with_a_lat_or_lng_as_none():
+    
+    # Arrange
+
+    location_type = t.LocationType(title='Location')
+
+    # Act
+
+    value = location_type.to_primitive(en.cv.Location(lat = 26.930101))
+
+    # Assert
+    eq_(value,{})
+
+@raises(DataError)
+def test_should_succeed_when_validate_location_raises_a_validation_error_when_passed_a_location_value_with_an_invalid_lat():
+    
+    # Arrange
+    class TestModel(MondayModel):
+        value = t.LocationType(id='location_1')
+    test = TestModel(id='item_id', name='Item Name')
+
+    # Act
+    test.value = en.cv.Location(-99,123.234)
+    test.validate()
+
+@raises(DataError)
+def test_should_succeed_when_validate_location_raises_a_validationerror_when_passed_a_location_value_with_an_invalid_lng():
+   
+   # Arrange
+    class TestModel(MondayModel):
+        value = t.LocationType(id='location_1')
+    test = TestModel(id='item_id', name='Item Name')
+
+    # Act
+    test.value = en.cv.Location(-23,223.234)
```

Comment:
Looks like this is being set to Lat instead of lng.  en.cv.Location(lng=-23,223.234) should work.